iotdb/src/test/java/org/apache/iotdb/db/engine/modification/DeletionQueryTest.java

```diff
@@ -276,6 +276,9 @@ public void testSuccessiveDeletion()
 
     FileNodeManager.getInstance().closeAll();
 
+    // wait for the file sync
+    Thread.sleep(3000);
```

Comment:
I do not like that ... wait for which file sync?

Reply:
OS file system, which does not always provide 100% visibility.